### ISSUE TITLE
Model Detectron2 for object detection

### ISSUE THREAD

**Issue by charaf90** [Ryudoro/Classification_aerienne#15]
Use the Detectron2 model to detect vehicles in images without separating them into individual vehicles. There should be two classes: Class 0 (no vehicle) and Class 1 (vehicle detected).

**Comment by Ryudoro** [comment]
tu pourrais rajouter une description ? et normalement il faut toujours mettre dans new d'abord !

**Comment by charaf90** [comment]
C'est fait

**Comment by Ryudoro** [comment]
Est-ce que ce modele renvoi egalement la position des vehicules ?

**Comment by charaf90** [comment]
Oui, on pourrait ensuite utiliser ces positions pour nos modèles de classifications.
